deno/deepgram-language-detection/README.md

```diff
@@ -0,0 +1,59 @@
+# ⚡ deepgramLanguageDetection()
+
+A Deno Cloud Function for detecting language from audio using [Deepgram](https://deepgram.com/)
+
+_Example function payload:_
+
+```json
+{
+  "fileUrl": "https://static.deepgram.com/examples/interview_speech-analytics.wav"
+}
+```
+
+_Successful function response::_
+
+```json
+{ "deepgramData": {}, "success": true }
+```
+
+_Error function response:_
+
+```json
+{ "message": "error message", "success": false }
+```
+
+## 📝 Variables
+
+List of variables used by this cloud function:
+
+**DEEPGRAM_API_KEY** - Your Deepgram API key.
+
+## 🚀 Deployment
+
+1. Clone this repository, and enter this function folder:
+
+```
+$ git clone https://github.com/open-runtimes/examples.git && cd examples
+$ cd deno/deepgram-transcribe-video
+```
+
+2. Enter this function folder and build the code:
+
+```
+docker run -e INTERNAL_RUNTIME_ENTRYPOINT=src/mod.ts --rm --interactive --tty --volume $PWD:/usr/code openruntimes/deno:v2-1.21 sh /usr/local/src/build.sh
+```
+
+As a result, a `code.tar.gz` file will be generated.
+
+3. Start the Open Runtime:
+
+```
+docker run -p 3000:3000 -e INTERNAL_RUNTIME_ENTRYPOINT=src/mod.ts -e INTERNAL_RUNTIME_KEY=secret-key --rm --interactive --tty --volume $PWD/code.tar.gz:/tmp/code.tar.gz:ro openruntimes/deno:v2-1.21 sh /usr/local/src/start.sh
```

Comment:
Maybe add a -d here to run the docker command detached

Reply:
Sorry I don't understand

Reply:
@gewenyu99 this might be fine considering most of the examples in the repo don't have the `-d` flag.

Reply:
@stnguyen90 👍 sure, I honestly think we should run all of them detached ;)

But I will accept this since all the other examples are like this.

Reply:
@gewenyu99, can this be resolved now?